install.sh

```diff
@@ -88,11 +81,24 @@ else
   exit 1
 fi
 
-LATEST_RELEASE="$(eval "${HTTP_GET} '${RELEASE_INFO_URL}'")"
+if [[ "${BUILDKITE_AGENT_VERSION:-"latest"}" == "latest" ]]; then
```

Comment:
(really nitpicky) If this were me I wouldn't bother with the inner quotes in the substitution:

```suggestion
if [[ "${BUILDKITE_AGENT_VERSION:-latest}" == "latest" ]]; then
```